editor/components/inserter/no-blocks.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+function NoBlocks() {
+	return (
+		<span className="editor-inserter__no-results">
+			{__('No blocks found')}
```

Comment:
There are probably some linting issues here. I recommend installing an eslint plugin for your editor to catch these errors soon. This should probably be `{ __('No blocks found') }`

Reply:
Editor configured hope not to repeat this again :)